src/ansys/aedt/core/hfss.py

```diff
@@ -1795,6 +1795,10 @@ def create_setup(self, name="MySetupAuto", setup_type=None, **kwargs):
         name = self.generate_unique_setup_name(name)
         setup = self._create_setup(name=name, setup_type=setup_type)
         setup.auto_update = False
+        if "Frequency" in kwargs.keys():
+            if type(kwargs["Frequency"]) is list:
+                if "MultipleAdaptiveFreqsSetup" not in kwargs.keys():
+                    kwargs["MultipleAdaptiveFreqsSetup"] = kwargs["Frequency"]
```

Comment:
```suggestion
        if "Frequency" in kwargs.keys() and type(kwargs["Frequency"]) is list:
            kwargs["MultipleAdaptiveFreqsSetup"] = kwargs["Frequency"]
```
I'm not sure if there is a need to check if `"MultipleAdaptiveFreqsSetup"` is in `kwargs`. Do we have a usecase where one defines `kwargs["Frequency"]` with a list and `kwargs["MultipleAdaptiveFreqsSetup"]` with a different one ?
Btw, what happens if the list is of length 1 ? Does the logic change from providing a single frequency value ?

Reply:
@SMoraisAnsys : Yes. This is a built-in keyword in the native API so we should proceed with caution. Passing ''**kwargs'' allows the user to pass any keywords that are valid in the native API.

Reply:
Will close. The proposed solution will throw an error if "Frequency" is not a key in kwargs.